sundar2021reproducibility/openreview/sections/methodology.tex

```diff
@@ -0,0 +1,29 @@
+\section{Methodology}
+
+The authors provide publicly accessible code\footnote{\href{https://github.com/google-research/rigl}{https://github.com/google-research/rigl}} written in Tensorflow (\citet{abadi2016tensorflow}). To gain a better understanding of various implementation aspects, we opt to replicate \textit{RigL} in Pytorch (\citet{Pytorch}). Our implementation extends the open-source code\footnote{\href{https://github.com/TimDettmers/sparse_learning}{https://github.com/TimDettmers/sparse\_learning}} of \citet{dettmers2020sparse} which uses a boolean mask to simulate unstructured sparsity. Our source code is publicly accessible on Github\footnote{\href{https://github.com/varun19299/rigl-reproducibility}{https://github.com/varun19299/rigl-reproducibility}} with training plots available on WandB\footnote{\href{https://wandb.ai/ml-reprod-2020}{https://wandb.ai/ml-reprod-2020}} (\citet{wandb}).
+
+
+\subsubsection{Mask Initialization}}For a network with $L$ layers and total parameters $N$, we associate each layer with a random boolean mask of sparsity $s_l, \; l \in [L]$. The overall sparsity of the network is given by $S=\frac{\sum_l s_l N_l}{N}$, where $N_l$ is the parameter count of layer $l$. Sparsities $s_l$ are determined by the one of the following mask initialization strategies:
```

Comment:
There is an extra `}` which is causing latex overflow of equation in page 9

Reply:
Fixed this in [commit](https://github.com/ReScience/RC-2020/pull/2/commits/d8f404ed4db5a7deccdb844d0d6ff6369b1f6255).